### PR TITLE
Smaller components

### PR DESCRIPTION
**Pull request by ablesable**

### CODE REVIEW

**Comment by ablesable** [comment]
It's all good man!

File: src/Pet.jsx [hunk 1]
```diff
@@ -9,22 +9,23 @@
...
+import { Link } from 'react-router-dom';
 
 const Pet = ({ name, animal, breed, images, location, id }) => {
   let hero = 'http://pets-images.dev-apis.com/pets/none.jpg';
   if (images.length) {
     hero = images[0];
   }
   return (
-    <a href={`/details/${id}`} className='pet'>
+    <Link href={`/details/${id}`} className='pet'>
```
Comment (MrBJasak) [inline src/Pet.jsx:20]: Save the href tags as constants in a separate file as a module. Save such things in the utils folder.

File: src/BreedSelect.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const BreedSelect = (props) => {
```
Comment (MrBJasak) [inline src/BreedSelect.jsx:1]: Use destructuring props.

Comment (MrBJasak) [inline src/BreedSelect.jsx:1]: First way: in first line on stary component make something like that const { arg1, arg2 } = props 
Second way: const BreedSelect = ({ arg1, arg2 }) => {

**Review by MrBJasak — COMMENTED** [review]
Just as Wołek said yesterday you can create a universal component name as InputSelect/Dropdown to use instead of using two other selectors. Pass the onChange functions as a prop

**Pull request merged**